spotinst-cli

```diff
@@ -393,11 +392,13 @@ for group, group_values in groups_to_update.iteritems():
         if options.scale_up is not None:
             command = "up"
             adjustment = options.scale_up
+            target = int(group_values[2]) + int(adjustment)
         elif options.scale_down is not None:
             command = "down"
             adjustment = options.scale_down
+            target = int(group_values[2]) - int(adjustment)
         # check that this change does not pass the percent_alert_threshold
-        ans = check_change_in_target(group, group_values[2], int(adjustment) + int(group_values[2]), percent_alert_threshold)
+        ans = check_change_in_target(group, group_values[2], target, percent_alert_threshold)
```

Comment:
I see that target is used only a single time, why use it as a variable?

Reply:
it is called twice, on lines 395, 399. it is differ by the ain order to show the right target amount on the text.

Reply:
It's not called twice, it's two different cases - only one is applicable at a time.
OK, I see your point: up until now, it only upscaled the group, right?

Reply:
no. the action was correct, but the message about the target to-be changed was wrong.